uber/utils.py

```diff
@@ -335,6 +335,10 @@ def from_sessionized_group(cls, d):
 
     @classmethod
     def from_sessionized_attendee(cls, d):
+        # Fix for attendees that were sessionized while the "requested_any_dept" column existed
+        if 'requested_any_dept' in d:
```

Comment:
Can we file an issue to remove this later?

Reply:
Absolutely! In fact the done condition should be to make from_sessionized more robust to handle the general case of unrecognized attributes.